components/squad/commons/squad_row.lua

```diff
@@ -63,41 +64,41 @@ function SquadRow:id(args)
 		error('Something is off with your input!')
 	end
 
-	local cell = mw.html.create('td')
+	local cell = Widget.TableCell{}
 	cell:addClass('ID')
 
 	local opponent = Opponent.resolve(
 		Opponent.readOpponentArgs(Table.merge(args, {type = Opponent.solo})),
 		nil, {syncPlayer = true}
 	)
-	cell:tag('b'):node(OpponentDisplay.InlineOpponent{opponent = opponent})
+	cell:addContent('<b>' .. tostring(OpponentDisplay.InlineOpponent{opponent = opponent}) .. '</b>')
```

Comment:
can't we use css styles on the cell instead so we can avoid the tostring and `<b></b>`?

or pass a html object to cell:addContent

Reply:
First wouldn't "be possible" as it's only part of the cell that has this style